### DISCUSSION TITLE
[FR]

### DISCUSSION THREAD

**Discussion by gabrielburcea** [ThinkR-open/golem#1034] [General]
Hello , 

I am trying to find the right way of adding the api key into golem shiny I am dev. 

This is an example 
Sys.setenv("CONNECT_SERVER"  = "https://rstudio-connect.adress.net")
 xwamexample - api key
Sys.setenv("CONNECT_API_KEY" = rstudioapi::askForPassword("Enter Connect Token:"))

Where do I pass this with RShiny golem? Do you have a specific way of doing this or you just place it into an RProfile inside the golem shiny app I am dev?

**Comment by ColinFay** [comment]
Hey @gabrielburcea 

Your q is a bit vague, what do you mean by "passing them to RShiny golem" ?

If you want to access the env var that is inside your global env, you can use Sys.getenv("CONNECT_API_KEY") anywhere in your app.

**Comment by gabrielburcea** [comment]
thanks, I have solved it. Just wanted to know if you have a specific way of doing this with golem. But nope, we should follow the generic setting.
